src/ssr-template-helper.js

```diff
@@ -105,10 +105,13 @@ export class SsrTemplateHelper {
         typeof data['html'] === 'string',
         'Server side html response must be defined'
       );
-      renderTemplatePromise = this.templates_.findAndSetHtmlForTemplate(
-        element,
-        /** @type {string} */ (data['html'])
-      );
+      renderTemplatePromise = this.viewer_.isTrustedViewer().then(trusted => {
```

Comment:
Should we also add a check to the `ssr()` method above?

Reply:
I don't think its strictly necessary, but its a good idea! Adding now